core/domain/feedback_services.py

```diff
@@ -241,21 +357,45 @@ def _get_thread_from_model(thread_model):
 
 
 def get_threads(exploration_id):
+    """Fetch all the threads for the given exploration id
+    Args:
+        exploration_id: str.
+
+    Returns:
+        list of FeedbackThread
```

Comment:
list of FeedbackThread. All threads corresponding to the given exploration id.

(IMO, "Returns" values should always have an explanation, since there is no variable name to make them self-evident.)